.github/workflows/release.yaml

```
@@ -0,0 +1,58 @@
+name: Release
+
+on:
+  workflow_dispatch: {}
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: true
+        fetch-depth: 0
+    - name: Set environment variables
+      run: ./build/ci/set_environment.sh
+    - name: Build chart
+      run: ./build/chart.sh
+      env:
+        IMAGE_REPOSITORY: ghcr.io/${{ github.repository }}
+    - name: Draft release
+      id: draft_release
+      uses: actions/create-release@v1
+      with:
+        release_name: ${{ env.GIT_TAG }}
+        tag_name: ${{ env.GIT_TAG }}
+        body: A MITS release.
+        draft: true
+        prerelease: false
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    - name: Upload chart to draft release
+      uses: actions/upload-release-asset@v1
+      with:
+        upload_url: ${{ steps.draft_release.outputs.upload_url }}
+        asset_path: output/mits-${{ env.VERSION }}.tgz
+        asset_name: mits-${{ env.VERSION }}.tgz
+        asset_content_type: application/x-gzip
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    - name: Build and publish image
+      uses: docker/build-push-action@v1
+      with:
+        registry: ghcr.io
+        repository: ${{ github.repository }}
+        tags: ${{ env.IMAGE_TAG }}
+        labels: >-
+          org.opencontainers.image.version=${{ env.VERSION }}
+        add_git_labels: true
+        always_pull: true
+        dockerfile: image/Dockerfile
+        username: ${{ secrets.GHCR_USERNAME }}
```

```suggestion
        username: ${{ github.repository_owner }}
```
([Reference](https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#github-context))

---

See my previous comment. Also, even if the `GITHUB_TOKEN` worked, `repository_owner` would be the organization once this was merged, which is an invalid user. GHCR can allow members to publish images, this is what we intend to do here.